src/login/logind-action.c

```diff
@@ -133,6 +133,44 @@ const HandleActionData* handle_action_lookup(HandleAction action) {
         return &handle_action_data_table[action];
 }
 
+static const HandleAction sleep_actions[] = {
```

Comment:
this is about the default, right? should probably be in the name.

i.e. default_sleep_actions

And maybe add a comment that this is about the order of preference.

Reply:
Not really (see my comment below) - this is the order in which we try each operations. Will add a comment.